cirq-core/cirq/devices/grid_device_metadata.py

```diff
@@ -40,6 +40,7 @@ def __init__(
         gateset: 'cirq.Gateset',
         gate_durations: Optional[Dict['cirq.GateFamily', 'cirq.Duration']] = None,
         all_qubits: Optional[Iterable['cirq.Qid']] = None,
+        compilation_target_gatesets: Optional[Iterable['cirq.CompilationTargetGateset']] = None,
```

Comment:
Either change the property return type to `Optional[Tuple['cirq.CompilationTargetGateset', ...]]` and return None if no target gateset is specified, or change the type of argument to `Iterable['cirq.CompilationTargetGateset']` with default value as an empty tuple `()`.

Reply:
Sg will change to the latter option. I had it this way to be consistent with other properties - cc @MichaelBroughton in case you have a preference as well.